core/capabilities/registry.go

```diff
@@ -199,3 +200,30 @@ func NewRegistry(lggr logger.Logger) *Registry {
 		lggr: lggr.Named("CapabilitiesRegistry"),
 	}
 }
+
+// TestMetadataRegistry is a test implementation of the metadataRegistry
```

Comment:
nit: I don't think it should be called "Test", maybe "Dummy" or "Default" or something similiar

Reply:
I agree with your sentiment. I had this discussion with @cedric-cordenier, too. Given that I am using it for testing at the moment and we don't have a clear product need, `Test` seems "calling it what it is".